pkgs/by-name/me/mealie/package.nix

```
@@ -0,0 +1,103 @@
+{ lib, callPackage, fetchFromGitHub, nixosTests, python3, python3Packages, writeShellScript }: let
+  version = "1.0.0-RC2";
```

Nit:

```suggestion
{ lib, callPackage, fetchFromGitHub, nixosTests, python3, python3Packages, writeShellScript }:

let
  version = "1.0.0-RC2";
```

There are no clear rules on this yet but this is pretty ugly and not done elsewhere in Nixpkgs.